ppo/agents/base_agent.py

```diff
@@ -53,17 +51,26 @@ def policy(policy_params: hk.Params, observation: np.ndarray):
 
         self.discount = discount
 
-    def select_action(self, observation: np.ndarray) -> np.ndarray:
+    def select_action_and_prob(self, observation: np.ndarray) -> Tuple[np.ndarray]:
         # Convert to get a batch shape
-        observation = tree_util.tree_map(
-            lambda x: jnp.expand_dims(x, axis=0), observation
-        )
+        observation = tree_util.tree_map(lambda x: jnp.expand_dims(x, axis=0), observation)
 
-        action = self.sampling_policy(
-            self.policy_params, next(self.sampling_keys), observation
-        )
+        action, log_prob = self.sampling_policy(self.policy_params, next(self.greedy_keys), observation)
```

Comment:
This needs to be changed :)
```suggestion
        action, log_prob = self.sampling_policy(self.policy_params, next(self.sampling_keys), observation)
```